ddtrace/_monkey.py

```diff
@@ -163,8 +163,18 @@ def on_import(hook):
                 "tracers", "integration_errors", 1, (("integration_name", module), ("error_type", type(e).__name__))
             )
         else:
+            version = ""
             imported_module.patch()
-            telemetry_writer.add_integration(module, True, PATCH_MODULES.get(module) is True, "")
+            if hasattr(imported_module, "get_version"):
```

Comment:
Can we avoid the ` hasattr` check? If we add `get_version` to every integration then this check should always pass. That way we have a consistent integrations api. Integrations that do not have a version (ex: logging) can return an empty string.